lib/benchmarks/client.dart

```diff
@@ -0,0 +1,9 @@
+import 'package:web_benchmarks/client.dart';
+
+import 'common.dart';
+
+// Run benchmarks.
```

Comment:
nit: this comment currently does not provide any new information over the code. Describe why/how the benchmarks are run in the main method of this program, and maybe how this program gets executed

Reply:
Thank you! I expanded the documentation.